src/app/init.js

```diff
@@ -1,16 +1,18 @@
 import 'polyfills' // should be first
 import '../styles/index.scss'
-import { createStore, applyMiddleware, combineReducers, compose as reduxCompose } from 'redux'
-import { reducer as form } from 'redux-form'
-import { createEpicMiddleware, combineEpics } from 'redux-observable'
+import API from './api'
+import { resourcesReducer } from 'ds-resource'
+import { cacheMiddleware, persistReducer } from 'ds-cache'
+import { thunkMiddleware, composeReducers, combineReducers } from 'ds-thunk'
```

Comment:
please rename thunkMiddleware. e.g. `promisableActionMiddleware` (in separate PR)

Reply:
https://github.com/django-stars/ds-thunk/pull/1

https://github.com/django-stars/frontend-skeleton/pull/205/commits/d442a62f8515aad2f7223742e4c96d95eee570f9

also i think it is better for now to rename ds-thunk as ds-redux-helpers and we will need to rename repo as well. I will update yarn lock later